core/src/toga/style/pack.py

```diff
@@ -867,33 +996,19 @@ def __css__(self) -> str:
         if self.height != NONE:
             css.append(f"height: {self.height}px;")
 
-        # alignment
```

Comment:
This CSS had its main and cross axes reversed, but it wasn't being caught because the tests were also flipped. Should be a moot point now.